actions/reconfigure.go

```diff
@@ -153,7 +153,7 @@ func (m *Reconfigure) reloadFromRegistry(addresses []string, instanceName, mode
 		var data map[string]interface{}
 		json.Unmarshal(body, &data)
 		count = len(data)
-		for key, _ := range data {
+		for key := range data {
```

Comment:
`range` returns two values that depend on the collection it uses. When `range` is used with a map, the first value is the key and the second is value. If one of the return values is not used, it should be specified as `_`. I'm surprised this change even compiled.